lib/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
-import io.deepmedia.tools.publisher.common.GithubScm
-import io.deepmedia.tools.publisher.common.License
-import io.deepmedia.tools.publisher.common.Release
-import io.deepmedia.tools.publisher.sonatype.Sonatype
+//import io.deepmedia.tools.publisher.common.GithubScm
```

Comment:
Instead of commenting, why don't we remove it.

Reply:
So that we can clone this project separately and just uncomment these in case we want to publish it